test/framework/module_generator.py

```diff
@@ -128,8 +128,8 @@ def test_descr(self):
             expected = '\n'.join([
                 'help([[%s]])' % gzip_txt,
                 '',
-                "whatis([[foo]])",
-                "whatis([[bar]])",
+                "whatis([[Description: foo]])",
+                "whatis([[Description: bar]])",
```

Comment:
the `Description: ` part is not there is `whatis` is specified, so change this back to what it was